ProjectToTest/Program.cs

```diff
@@ -1,205 +1,71 @@
-namespace HOMEWORK3_QUADRATIC_YPARKHOMENKO
+using ProjectToTest;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+internal class Program
 {
-    internal class Program
+    public static bool IsValidName(string name = "")
```

Comment:
A method with default value? Why??? Actually default value of string is `null` I suppose. Why to initialize it?